pallets/migrations/Cargo.toml

```diff
@@ -29,3 +29,4 @@ std = [
 	"sp-core/std",
 	"environmental/std",
 ]
+try-runtime = ["frame-support/try-runtime"]
```

Comment:
We'll need to add this pallet's `try-runtime` feature to any runtime's that use the pallet too.

Reply:
Good catch

Reply:
e58599f